src/energy_comms/transform/cps.py

```diff
@@ -0,0 +1,50 @@
+# imports
+
+import json
+
+import pandas as pd
+import prettytable
+import requests
+
+
+def transform(x):
+    # convert API pretty table to pandas dataframe
+
+    tbl_as_csv = x.get_csv_string().replace("\r", "")
+
+    txt = open("national_unemployment.csv", "w")
+
+    n = txt.write(tbl_as_csv)
+    txt.close()
+
+    cps = pd.read_csv("national_unemployment.csv")
+
+    annual_average = cps.groupby("year").agg({"value": "mean"}).reset_index()
+
+    # round to match official sig figures on BLS website
+
+    annual_average = annual_average.round(1)
+
+    # manually input missing years data from:
```

Comment:
Confused what's happening here. More commenting to explain the situation could be good.